packages/react-components/src/organisms/DiscussionModal.tsx

```diff
@@ -83,8 +98,8 @@ const QuickCheckReplyModal: React.FC<QuickCheckReplyModalProps> = ({
     handleSubmit,
   } = methods;
 
-  const onSubmit = async (data: QuickCheckReplyModalData) => {
-    await onReplyToDiscussion(discussionId, data);
+  const onSubmit = async (data: DiscussionModalData) => {
+    await onSave(discussionId, data);
```

Comment:
if we change this `DiscussionModalData` to be `{ text: string }`

```suggestion
    await onSave(discussionId, data.text);
```